README.md

```diff
@@ -1,2 +1,43 @@
-# trustymail
-Scan domains and return data based on trustworthy email best practices
+## Trustworthy Mail  :lock:
+`TrustyMail` is a tool that evaluates SPF/DMARC DNS record configurations to recommend best practices for the records. It saves its results to a CSV.
+
+#### Usage and examples
+
+```bash
+python cli [options] INPUT
```

Comment:
Our instructions should feature running trustymail post-installation (as a module), à la invoking `trustymail` directly. We can also mention running directly, like we have with pshtt. https://github.com/dhs-ncats/pshtt#getting-started